File: amplify/backend/function/mserviceFunction/src/py-template/mservice_main.py
import __main__ as userMain
import os

# os.environ['INPUT_MESSAGE_TOPIC']
# os.environ['OUTPUT_MESSAGE_TOPIC']
# event['sender']

def handler(event, context):
    if ('AIOT_RUN_FROM_EDGE' in os.environ) and (os.environ['AIOT_RUN_FROM_EDGE'] == 'true'):
        # do nothing
        pass
    else:
        import aiothings as aiot
        if (os.environ['INPUT_MESSAGE_TOPIC'] == 'null') and (os.environ['OUTPUT_MESSAGE_TOPIC'] == 'null'):
            # do nothing
            pass
        else:
            aiot.setInput(event)
    # aiot.consoleOutput('Console display')   # Output to console
    return userMain.__handler__(event, context)

File: amplify/backend/function/mserviceFunction/src/js-template/mservice_main.js
let aiot = null
const userMain = require('./__main__');

// process.env.INPUT_MESSAGE_TOPIC
// process.env.OUTPUT_MESSAGE_TOPIC
// event.sender

exports.handler = (event, context, callback) => {
    if (process.env.hasOwnProperty('AIOT_RUN_FROM_EDGE') && process.env.AIOT_RUN_FROM_EDGE === 'true') {
        // do nothing
    } else {
        aiot = require('./aiothings');
        if (process.env.INPUT_MESSAGE_TOPIC === 'null' && process.env.OUTPUT_MESSAGE_TOPIC === 'null') {
            // do nothing
        } else {
            aiot.setInput(event);
        }
    }
    // Add your code here
    // aiot.consoleOutput('main:' + process.env.MSERVICE_NAME); // display from console
    userMain.__handler__(event, context, callback)
};